tests/server/config/config_test.go

```diff
@@ -35,36 +35,36 @@ var dialClient = &http.Client{
 	},
 }
 
-func Test(t *testing.T) {
-	TestingT(t)
-}
-
-var _ = Suite(&testConfigPresistSuite{})
+type configPersistTestSuite struct {
+	suite.Suite
 
-type testConfigPresistSuite struct {
 	cleanup func()
 	cluster *tests.TestCluster
 }
 
-func (s *testConfigPresistSuite) SetUpSuite(c *C) {
+func TestConfigPersistTestSuite(t *testing.T) {
+	suite.Run(t, new(configPersistTestSuite))
+}
+
+func (suite *configPersistTestSuite) SetupSuite() {
 	ctx, cancel := context.WithCancel(context.Background())
-	s.cleanup = cancel
+	suite.cleanup = cancel
 	cluster, err := tests.NewTestCluster(ctx, 3)
-	c.Assert(err, IsNil)
-	c.Assert(cluster.RunInitialServers(), IsNil)
-	c.Assert(cluster.WaitLeader(), Not(HasLen), 0)
-	s.cluster = cluster
+	suite.NoError(err)
+	suite.NoError(cluster.RunInitialServers())
+	suite.NotEmpty(cluster.WaitLeader())
+	suite.cluster = cluster
 }
 
-func (s *testConfigPresistSuite) TearDownSuite(c *C) {
-	s.cleanup()
-	s.cluster.Destroy()
+func (suite *configPersistTestSuite) TearDownSuite() {
+	suite.cleanup()
+	suite.cluster.Destroy()
 }
 
-func (s *testConfigPresistSuite) TestRateLimitConfigReload(c *C) {
-	leader := s.cluster.GetServer(s.cluster.GetLeader())
+func (suite *configPersistTestSuite) TestRateLimitConfigReload() {
```

Comment:
It seems that we only have one test for `configPersistTestSuite`, so there is no need to use suite here?

Reply:
I'm not sure if other tests will be added to this suite. cc @CabinfeverB

Reply:
There are no planned tests to be added